packages/story-editor/src/components/canvas/karma/carousel.karma.js

```diff
@@ -74,10 +75,14 @@ describe('Carousel integration', () => {
   }
 
   async function clickOnThumbnail(index) {
-    await fixture.editor.footer.carousel.waitReady();
-    const thumb = fixture.editor.footer.carousel.pages[index];
-    thumb.node.scrollIntoView();
-    await fixture.events.mouse.clickOn(thumb.node, 5, 5);
+    await fixture.events.sleep(100);
+    const { pages } = await fixture.editor.footer.carousel;
+    // node is found fine running locally while _node is present when running remote.
+    const page = await pages[index];
+    const thumb = !page?.node ? page._node : page.node;
```

Comment:
That is odd 🤔 

@barklund Do you perhaps know what's up there?

I'd want to avoid checks like that wherever possible, so curious to know what's going on here.

Reply:
yeah odd right? i wanted to see if this solves the failures. this is what a `PageThumb` looks like when run remote: 
```
PageThumb({
      _node: (
        <button
          aria-label="Page 1 (current page)"
          tabindex="0"
          role="option"
          data-page-id="page1"
          width="40"
          height="60"
          class="pagepreview__Page-sc-i45m4c-0 dGxJop"
        >
          ...
        </button>
      ),
      _path: 'editor.footer.carousel.pages[page1]',
    }),
```

Reply:
anyways if this next run works i'll continue down this path to figure out the discrepancy and remove the ternary.

Reply:
`_node` and `_path` are "internal" class properties on the `Container` class (`packages/story-editor/src/karma/fixture/containers/container.js`).

`node` and `path` are just getters for `_node` and `_path`. So `PageThumb.node` _should_ actually just return `_node`.

Are you saying that `page.node` is `null`/`undefined`?

Reply:
yeah, they were at least before i added the sleep.  wanted to see if this allowed the tests to be successful. it did so now it's time to remove and make it work consistently. fear not 🙃  it won't stay. it just only fails remotely so i have to keep pushing up to try things.